template/dockerfiles/gomodule/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM golang:{{VERSION}}
+FROM golang:{{VERSION}} AS builder
 ENV PORT {{PORT}}
 EXPOSE {{PORT}}
 
-WORKDIR /go/src/app
+WORKDIR /build
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
 COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o app-binary
 
-RUN go get
-RUN go build -v -o app
-RUN mv ./app /go/bin/
-
-CMD ["app"]
\ No newline at end of file
+FROM scratch
```

Comment:
this is a breaking change. the golang base image includes debian bundled with some other tooling. Scratch does not.

many applications depend on having underlying OS features. Given Draft's purpose I'm not sure we should limit that.

I propose we use alpine or if we really want to go slim, distroless.

Reply:
alpine sounds good to keep the size small without removing all the tooling